src/states/node.rs

```diff
@@ -1247,20 +1294,18 @@ impl Node {
                 if let Err(error) = self.send_direct_message(peer_id, direct_message) {
                     debug!("{:?} failed requesting resource_proof from node candidate {:?}/{:?}.",
                            self,
-                           public_id.name(),
+                           name,
                            error);
                 }
             }
-            Ok(None) => {
-                debug!("{:?} adding {:?} into routing table.",
-                       self,
-                       public_id.name());
+            Ok(false) => {
+                debug!("{:?} adding {:?} into routing table.", self, name);
```

Comment:
I'm not sure this log statement isn't just spam - we print inside `add_to_routing_table()` too.